Sources/Core/AWSSDKEventStreamsAuth/AWSMessageSigner.swift

```diff
@@ -5,13 +5,22 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-import ClientRuntime
+import struct AWSSDKHTTPAuth.AWSSigningConfig
+import protocol SmithyEventStreamsAPI.MessageEncoder
+import struct SmithyEventStreamsAPI.Message
+import protocol SmithyEventStreamsAuthAPI.MessageSigner
+import protocol SmithyEventStreamsAuthAPI.MessageDataSigner
+import protocol SmithyHTTPAuthAPI.Signer
+import struct Smithy.Attributes
+import struct Smithy.AttributeKey
+
+// Code left indented to prevent Git diff from being blown up by whitespace changes.
+// Will fix after event stream modularizaion has been reviewed.
 
-extension AWSEventStream {
     /// Signs a `Message` using the AWS SigV4 signing algorithm
     public class AWSMessageSigner: MessageSigner {
```

Comment:
This is the SigV4 signer for event streams.  It is moved from `AWSClientRuntime` to the new `AWSSDKEventStreamsAuth` module.